pkg/webui/account/views/oauth-client-overview/index.js

```diff
@@ -0,0 +1,173 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { useIntl } from 'react-intl'
+import { Col, Row, Container } from 'react-grid-system'
+
+import applicationIcon from '@assets/misc/application.svg'
+
+import DataSheet from '@ttn-lw/components/data-sheet'
+
+import IntlHelmet from '@ttn-lw/lib/components/intl-helmet'
+import DateTime from '@ttn-lw/lib/components/date-time'
+import withRequest from '@ttn-lw/lib/components/with-request'
+
+import EntityTitleSection from '@console/components/entity-title-section'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import { mayPerformAdminActions } from '@account/lib/feature-checks'
+
+import { getCollaboratorsList } from '@account/store/actions/collaborators'
+
+import {
+  selectSelectedClient,
+  selectSelectedClientId,
+  selectClientFetching,
+} from '@account/store/selectors/clients'
+import { selectCollaboratorsTotalCount } from '@account/store/selectors/collaborators'
+
+import style from './oauth-client-overview.styl'
+
+const { Content } = EntityTitleSection
+
+const capitalize = str => str.charAt(0).toUpperCase() + str.slice(1)
```

Comment:
I think we use this now in a couple of places, so let's move this into it's own `/lib` module.